docs/lang/articles/reference.md

```diff
@@ -250,8 +250,56 @@ with `expression2` as the error message.
 
 ## Compound statements
 
+This section describes the supported syntax of compound statements in Taichi,
+and it is modeled after [the compound statement chapter of The Python Language Reference](https://docs.python.org/3/reference/compound_stmts.html)
```

Comment:
The "model after" part is already stated in introduction. We can directly say "This section explains the syntax and semantics of compound statements in Taichi".